components/encumbered/dav1d/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2021 (c) Tim Mooney. All rights reserved.
+#
+BUILD_BITS=		64_and_32
+BUILD_STYLE=		meson
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		dav1d
+COMPONENT_VERSION=	0.9.2
+COMPONENT_FMRI=		codec/dav1d
+COMPONENT_SUMMARY=	A fast, cross-platform AV1 decoder
+COMPONENT_CLASSIFICATION=System/Multimedia Libraries
+COMPONENT_PROJECT_URL=	https://code.videolan.org/videolan/dav1d
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL= \
+	$(COMPONENT_PROJECT_URL)/-/archive/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:59a5fc9cc5d8ea780ad71ede6d589ed33fb5179d87780dcf80a00ee854952935
+COMPONENT_LICENSE=	BSD 2-Clause Simplified License
+
+include $(WS_MAKE_RULES)/encumbered.mk
+include $(WS_MAKE_RULES)/common.mk
+
+CONFIGURE_OPTIONS += -Denable-asm=true
+
```

Comment:
Build fails:
meson.build:25:0: ERROR: Unknown options: "enable-asm"

You have misspelled the option (correct: enable_asm), but it's default is true anyways.

Reply:
Thanks for pointing out the incorrect option name, I'll fix that.  However for me, it's only a warning and does not stop the build.  Do you have some additional "meson" flags in your environment that convert warnings to errors? 

Here's a small snippet from a build log where it warns, and then continues with the `setup` process:

```
The Meson build system
Version: 0.59.3
Source dir: /export/home/mooney/oi-userland/components/encumbered/dav1d/dav1d-0.9.2
Build dir: /export/home/mooney/oi-userland/components/encumbered/dav1d/build/i86
Build type: native build
WARNING: Unknown options: "enable-asm"
The value of new options can be set with:
meson setup <builddir> --reconfigure -Dnew_option=new_value ...
Project name: dav1d
Project version: 0.9.2
C compiler for the host machine: /usr/gcc/7/bin/gcc (gcc 7.5.0 "gcc (OpenIndiana 7.5.0-il-0) 7.5.0")
C linker for the host machine: /usr/gcc/7/bin/gcc ld.solaris 5.11-1.1771 (illumos)
Host machine cpu family: x86
Host machine cpu: i686
Run-time dependency threads found: YES
Checking for function "clock_gettime" : YES 
Library dl found: YES
Checking for function "dlsym" with dependency -ldl: YES 
```

The option name will be fixed in my next commit.